web-console/src/lib/components/streaming/import/randomData/index.tsx

```diff
@@ -43,14 +45,14 @@ export const RngFieldSettings = (props: {
     return {
       bool_const: false,
       true_pct: '0.5',
-      value: '0',
-      min: '0',
-      max: '100',
-      mu: '100',
-      sigma: '10',
-      lambda: '0.1',
-      alpha: '0.1',
-      beta: '0.1',
+      value: new BigNumber('0'),
```

Comment:
It looks like now everything is converted to BigNumber.. which is probably fine but it seems like now we need to protect the types that of all ranges correctly (e.g., if it's 64bit we should not be able to have something larger) I didin't see the code yet where this is enforced